tests/bluetooth/controller/uut/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# CMakeLists.txt file for creating of uut library.
+#
+
+add_library(uut STATIC
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/util/mem.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/util/memq.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_tx_queue.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_conn.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ll_feat.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_cc.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_pdu.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_remote.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_local.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_enc.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_common.c
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_llcp_chmu.c
+)
+
+target_include_directories(uut PUBLIC
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/include
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/common/include
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/include
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/nordic
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth
```

Comment:
FYI: Those include paths are also defined by the `mocks` library, which in practice means you can remove them from here.

But as the `uut` library refers to source files directly in the bluetooth controller library then it also make sense to keep those includes explicitly here, as well as in the `mocks` lib.

Reply:
I prefer listing them here explicitly, due to the direct reference to bluetooth controller files

Reply:
The consequence is however that we also need to leave the  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/include in place to ensure that we include the mock include-files.
i.e. either we remove all include paths, or we leave all of them as is